src/types/storeTypes.ts

```diff
@@ -14,9 +14,9 @@ export interface AppState {
 
     // We keep all data in dictionaries with country ids as keys, mirroring the resources on disk
     admin1Indicators: Dict<FeatureIndicators>;
-    admin1Geojson: Dict<Geojson>;
+    admin1Geojson: Dict<Feature<Polygon | MultiPolygon>[]>;
     admin2Indicators: Dict<FeatureIndicators>;
-    admin2Geojson: Dict<Geojson>;
-    admin0GeojsonFeature: Feature | null;
+    admin2Geojson: Dict<Feature<Polygon | MultiPolygon>[]>;
+    admin0GeojsonFeature: Feature<Polygon | MultiPolygon> | null;
```

Comment:
Might be terser to update the `Geojson` type in `resourceTypes` to `Feature<Polygon | MultiPolygon>`..?

Reply:
types are updated in big refactor branch instead but have taken on this suggestion!